### PR TITLE
Feature adding keywords to searchbar

### PR DESCRIPTION
**Pull request by william-jensen01**
Added keywords for 'students' and 'teachers' to searchbar. I included some comments in my code to explain what is going on. Let me know if you have any questions!

### CODE REVIEW

**Comment by william-jensen01** [comment]
[add keywords to search for "student" and "mentor"](https://trello.com/c/bfGvdy3L/28-add-keywords-to-search-for-student-and-mentor)

**Review by LittleBoGeek — APPROVED** [review]
Looks good to me. The commits are nice and small and you did a nice job of commenting your code.

File: src/components/search-bar/SearchBarAutoComplete.js [hunk 1]
```diff
@@ -123,14 +124,44 @@ const SearchBarAutoComplete = () => {
...
+      setNewListOptions([listOptions[1]]); // location of 'teachers' object in the listOptions array
+      setFilter(false);
+      return newListOptions;
+    }
+    else {
+      setNewListOptions(listOptions); // go back to the original listOptions
+      setFilter(true);
+      return newListOptions;
+    }
+  };
```
Comment (PaulMEdwards) [inline src/components/search-bar/SearchBarAutoComplete.js:156]: Pretty clever solution; I like it!

**Review by PaulMEdwards — APPROVED** [review]
I agree with Nadia regarding the conciseness of your thorough comments.

**Pull request merged**